lib/SearchAndSort/SearchAndSort.js

```diff
@@ -1058,7 +1060,7 @@ class SearchAndSort extends React.Component {
           disabled={!searchTerm}
           data-test-search-and-sort-submit
         >
-          <FormattedMessage id="stripes-smart-components.search" />
+          <FormattedMessage id={this.props.searchFiledButtonLabel || 'stripes-smart-components.search'} />
```

Comment:
All other `this.props` values are destructured above beginning on line 1008/1010. Please follow the established convention. And as above, fix the typo: `s/Filed/Field/`.

Reply:
Done.